zebra-consensus/src/transaction.rs

```diff
@@ -63,7 +63,7 @@ where
 ///
 /// Transaction verification has slightly different consensus rules, depending on
 /// whether the transaction is to be included in a block on in the mempool.
-#[allow(dead_code)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
Style nit

```suggestion
#[derive(Clone, Debug, Eq, PartialEq)]
```

Reply:
We need a clippy lint or auto-fix for this